app/controllers/restaurant_controller.rb

```diff
@@ -2,7 +2,12 @@ class RestaurantController < ApplicationController
 
   def show
     service = RevItUpService.new(current_user.lat, current_user.lng, meters_to_miles, params["price"])
-    @restaurant = RevItUpFacade.new(service.restaurant)
+    if service.restaurant == "error"
+      flash[:notice] = "No Restaurants Meet This Criteria, Please Try Again"
```

Comment:
Minor styling change; I do not think we need to make the sentence fully capitalized.

Reply:
agreed